vmdb/app/controllers/api_controller/vms.rb

```diff
@@ -85,5 +96,13 @@ def suspend_vm(vm, klass)
     rescue => err
       action_result(false, err.to_s)
     end
+
+    def destroy_vm(vm)
+      desc = "#{vm_ident(vm)} deleting"
+      task_id = queue_object_action(vm, desc, :method_name => "destroy")
```

Comment:
@abellotti I just realized that this ignores the existing `_queue` methods that are already defined on the models.  Is that intentional?